setup.py

```diff
@@ -45,9 +45,10 @@
     extras_require={
         'dev': ['pytest', 'pytest-cov', 'sphinx', 'pandas', 'matplotlib'],
     },
-    # package_data={
-    #     'sample': ['package_data.dat'],
-    # },
+    include_pacakge_data=True,
```

Comment:
Thanks, fixed in https://github.com/pystitch/stitch/commit/e4ea047ba53ec50cb40d7817e0330eda7c72ac06